From c04bee088f9446d3e06db0928ce410748bc3a25c Mon Sep 17 00:00:00 2001
From: Nicolas Ferre <nicolas.ferre@atmel.com>
Date: Wed, 3 Jul 2013 12:51:36 +0200
Subject: [PATCH 328/335] ARM: at91/DT: at91sam9x5ek: fix USB host property to
 enable port C

Device Tree "num-ports" property of USB host node has to be
set to maximum number of ports available.
The possibility to activate a particular port is done by specifying the proper
gpio configuration for its vbus.

This patch fixes the USB host node by configuring the 3 ports available on the
product and letting "port A" available for USB gadget usage.

Reported-by: Rodolfo Giometti <giometti@enneenne.com>
Signed-off-by: Nicolas Ferre <nicolas.ferre@atmel.com>
Acked-by: Jean-Christophe PLAGNIOL-VILLARD <plagnioj@jcrosoft.com>
Cc: Bo Shen <voice.shen@atmel.com>

Conflicts:
	arch/arm/boot/dts/at91sam9x5ek.dtsi
---
 arch/arm/boot/dts/at91sam9x5ek.dtsi |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/at91sam9x5ek.dtsi b/arch/arm/boot/dts/at91sam9x5ek.dtsi
index 620ec3e..5791a0c 100644
--- a/arch/arm/boot/dts/at91sam9x5ek.dtsi
+++ b/arch/arm/boot/dts/at91sam9x5ek.dtsi
@@ -94,8 +94,9 @@
 
 		usb0: ohci@00600000 {
 			status = "okay";
-			num-ports = <2>;
-			atmel,vbus-gpio = <&pioD 19 1
+			num-ports = <3>;
+			atmel,vbus-gpio = <0 /* &pioD 18 1 *//* Activate to have access to port A */
+					   &pioD 19 1
 					   &pioD 20 1
 					  >;
 		};
-- 
1.7.10.4

